### ISSUE TITLE
Building the vignette fails at pk_anatomical_detail()

### ISSUE THREAD

**Issue by hlapp** [xu-hong/rphenoscape#8]
Here's the corresponding output:

```
* creating vignettes ... ERROR
Warning in .nextMethod(obj = obj, from = from) :
  NAs introduced by coercion
....This might take a while....
https://scholar.google.com/scholar?q=The+Phylogeny+of+Ictalurid+Catfishes%3A+A+Synthesis+of+Recent+Work&btnG=&hl=en&as_sdt=0%2C42
Parse NeXML....
Map symbols to labels...
Quitting from lines 106-107 (rphenoscape.Rmd) 
Error: processing vignette 'rphenoscape.Rmd' failed with diagnostics:
Each variable must be a 1d atomic vector or list.
Problem variables: relationships.
Execution halted
Error: Command failed (1)
Execution halted
```

The respective code chunk is the following:

```R
pk_anatomical_detail("basihyal bone")
```

I suspect that's related to the test failure reported in #6.

**Comment by hlapp** [comment]
I think I found the issue for this, see phenoscape/phenoscape-kb-services#43. In essence, it looks like the return value for the term detail API endpoint was changed to return an array of relationships as well, which won't work with the conversion to a `data.frame`. 

I suppose for now the `relationships` attribute just needs to be removed from the JSON or the resulting list before handing it over to `dplyr::as_data_frame(lst)`.
